pkg/url/types.go

```diff
@@ -52,6 +52,14 @@ const (
 	StateTypeLocallyDeleted = "Locally Deleted"
 )
 
+func NewURLsFromKubernetesIngressList(kil *unions.KubernetesIngressList) []URL {
+	var ul []URL
+	for _, it := range kil.Items {
+		ul = append(ul, NewURLFromKubernetesIngress(it))
```

Comment:
if `NewURLFromKubernetesIngress()` returns ` URL{}`, we should `continue` instead of adding the empty ingress to the list.

Reply:
done